src/py/mat3ra/made/tools/build/interface/builders.py

```diff
@@ -270,36 +272,74 @@ class CommensurateLatticeTwistedInterfaceBuilder(BaseBuilder):
     def _generate(self, configuration: _ConfigurationType) -> List[_GeneratedItemType]:
         film = configuration.film
         # substrate = configuration.substrate
-        max_search = self.build_parameters.max_repetition_int
         a = film.lattice.vector_arrays[0][:2]
         b = film.lattice.vector_arrays[1][:2]
-        commensurate_lattice_pairs = self.__generate_commensurate_lattices(
-            configuration, a, b, max_search, configuration.twist_angle
+        max_int = self.build_parameters.max_repetition_int or self.__get_initial_guess_for_max_int(
+            film, configuration.twist_angle
         )
+        commensurate_lattice_pairs: List[CommensurateLatticePair] = []
+        while not commensurate_lattice_pairs and max_int < 42:
```

Comment:
Should be configurable as an optional builder parameter

Reply:
Well, we only want to limit infinite loop. This will stop for up to angle of ~0.8 degrees